.ci/install_crio.sh

```diff
@@ -112,9 +112,11 @@ crio_config_file="/etc/crio/crio.conf"
 # This commit contains changes regarding CRI-O’s default configuration files,
 # so we have to use a new default config path from here.
 # https://github.com/cri-o/cri-o/commit/0f1226b99685f95e83c94dc6668b6452df5056db
+pushd "${GOPATH}/src/${crio_repo}"
 if git merge-base --is-ancestor 0f1226b99685f95e83c94dc6668b6452df5056db HEAD; then
```

Comment:
@haircommander @saschagrunert do you know if this is still valid? I see that crio config file is still being installed as `/etc/crio/crio.conf`

Reply:
@chavafg, a `make && make install` would install `crio.conf` in `/etc/crio/crio.conf`, but would create for you `/etc/crio/crio.conf.d` directoy where you could drop the snippets you want.

What's the problem you're facing, Salvador?

Reply:
@fidencio, this code was added some time ago as it seems that at some point the cri-o config file changed to `/etc/crio/crio.conf.d/00-default.conf`. Right now Gaby is trying to fix #2959, but it seems that this code is no longer valid as it is indeed installing the default config file in `/etc/crio/crio.conf`

@GabyCT it should be fine deleting this lines (112 - 119)

Reply:
@chavafg ok let me do it thanks

Reply:
@chavafg, that behaviour was reverted as part of https://github.com/cri-o/cri-o/commit/7f4ac3b7d7c7dc126bc1ca53bc35b59b97bb21dc

@GabyCT, please, just drop that piece as suggested by @chavafg.

Reply:
thanks